assets/prototype/domain/shared/entities/tickets/predicates/isValidEntityOrArchive.ts

```diff
@@ -1,10 +1,10 @@
 import { Ticket } from '../../../../eventEditor/data/types';
-import isTrashed from './isTrashed';
+import isTrashed from '../../../predicates/shared/isTrashed';
 
 /**
  * @function
  * @param {Object} ticket model object
- * @param {boolean} includeArchived if true will not filter out trashed entities
+ * @param {boolean} includeTrashed if true will not filter out trashed entities
  * @return {boolean} true if event date is valid entity or archive
  */
 const isValidEntityOrArchive = (ticket: Ticket, includeArchived: boolean): boolean => {
```

Comment:
```suggestion
const isValidOrTrashed = (ticket: Ticket, includeTrashed: boolean): boolean => {
```

not sure if that name is that great...

maybe:

- `conditionalTrashFilter`
- `isMaybeTrashed`
- `isNotOrIsTrashed`
- `isNotTrashedOrIsTrashedButThatsOkay` 😆

???

Reply:
+1 for `isNotTrashedOrIsTrashedButThatsOkay` :smiley: